src/ripple/app/tests/HashRouter_test.cpp

```diff
@@ -171,14 +181,10 @@ class HashRouter_test : public beast::unit_test::suite
     void
     run()
     {
-        UptimeTimer::getInstance().beginManualUpdates();
-
         testExpiration();
         testSuppression();
         testSetFlags();
         testSwapSet();
-
-        UptimeTimer::getInstance().endManualUpdates();
     }
 };
 
```

Comment:
I don't see a test for the `touch` change of behaviour. The comments do reflect the new `touch` time, but I'd like to see an `expect` that now gets a flag that would expire under the old algorithm. (and if there is such a test, I missed it.)